doc/radosgw/layout.rst

```diff
@@ -178,22 +183,22 @@ Known pools:
   prodtx$prodt.buckets
   test2
 
-.users.email
+  namespace: users.email
```

Comment:
New version pushed, also fixing some pre-existing formatting issues. There's still room for further improvement, but I think this should work for now.